### PR TITLE
Upgrade Armeria to 0.55.1

### PR DESCRIPTION
**Pull request by huydx**

### CODE REVIEW

**Comment by imasahiro** [comment]
I already saw some PRs to fix defect at armeria. https://github.com/line/armeria/issues?q=label%3Adefect+milestone%3A0.55.1+is%3Aopen
Better to wait 0.55.1?

**Comment by huydx** [comment]
@imasahiro Sound good to me, just take this PR as place holder, and update the detail number later

**Comment by trustin** [comment]
@huydx 0.55.1 is out. Please update. :-)

File: server/src/main/java/com/linecorp/centraldogma/server/internal/admin/service/TokenService.java [hunk 1]
```diff
@@ -111,7 +111,7 @@ public TokenService(ProjectManager projectManager, CommandExecutor executor) {
             if (exist) {
                 // TODO(hyangtack) Would send 409 conflict when the following PR is merged.
                 // https://github.com/line/armeria/pull/746
-                throw new HttpResponseException(HttpStatus.CONFLICT);
+                throw HttpResponseException.of(HttpStatus.CONFLICT);
```
Comment (trustin) [inline server/src/main/java/com/linecorp/centraldogma/server/internal/admin/service/TokenService.java]: Please use `HttpStatusException`.

Comment (hyangtack) [inline server/src/main/java/com/linecorp/centraldogma/server/internal/admin/service/TokenService.java]: @huydx Would you change this to `HttpStatusException.of(HttpStatus.CONFLICT)` in order to reuse a cached exception object? :-)

**Comment by minwoox** [comment]
Since this is the upgrade of the dependency, would you change these as well please?

- Checkstyle 8.4 -> 8.5
- Guava 23.4 -> 23.5

**Comment by trustin** [comment]
You got a compilation error: https://travis-ci.org/line/centraldogma/builds/309330623#L586

**Comment by trustin** [comment]
You'll need this fix:

```diff
diff --git a/server/src/test/java/com/linecorp/centraldogma/server/internal/admin/decorator/DecoratorTest.java b/server/src/test/java/com/linecorp/centraldogma/server/internal/admin/decorator/DecoratorTest.java
index 2e26adc..8b8c96d 100644
--- a/server/src/test/java/com/linecorp/centraldogma/server/internal/admin/decorator/DecoratorTest.java
+++ b/server/src/test/java/com/linecorp/centraldogma/server/internal/admin/decorator/DecoratorTest.java
@@ -40,7 +40,7 @@ import com.linecorp.armeria.common.HttpStatus;
 import com.linecorp.armeria.common.RequestContext;
 import com.linecorp.armeria.common.util.SafeCloseable;
 import com.linecorp.armeria.internal.DefaultAttributeMap;
-import com.linecorp.armeria.server.HttpResponseException;
+import com.linecorp.armeria.server.HttpStatusException;
 import com.linecorp.armeria.server.Service;
 import com.linecorp.armeria.server.ServiceRequestContext;
 import com.linecorp.centraldogma.server.internal.admin.authentication.User;
@@ -112,9 +112,9 @@ public class DecoratorTest {
             assertThat(new ProjectMembersOnly().serve(delegate, ctx, null).aggregate().join().status())
                     .isEqualTo(HttpStatus.OK);
             assertThatThrownBy(() -> new ProjectOwnersOnly().serve(delegate, ctx, null))
-                    .isInstanceOf(HttpResponseException.class)
+                    .isInstanceOf(HttpStatusException.class)
                     .satisfies(cause -> {
-                        assertThat(((HttpResponseException) cause).httpStatus())
+                        assertThat(((HttpStatusException) cause).httpStatus())
                                 .isEqualTo(HttpStatus.UNAUTHORIZED);
                     });
 
```

**Comment by trustin** [comment]
Here's another fix:
```diff
diff --git a/server/src/test/java/com/linecorp/centraldogma/server/internal/admin/decorator/DecoratorTest.java b/server/src/test/java/com/linecorp/centraldogma/server/internal/admin/decorator/DecoratorTest.java
index 7ce04a9..6d705c0 100644
--- a/server/src/test/java/com/linecorp/centraldogma/server/internal/admin/decorator/DecoratorTest.java
+++ b/server/src/test/java/com/linecorp/centraldogma/server/internal/admin/decorator/DecoratorTest.java
@@ -92,43 +92,47 @@ public class DecoratorTest {
 
     @Test
     public void testProjectAccessController() throws Exception {
-        final EventLoop ev = spy(mock(EventLoop.class));
-        final ServiceRequestContext ctx = spy(mock(ServiceRequestContext.class));
-        when(ctx.eventLoop()).thenReturn(ev);
+        final EventLoop ev = new DefaultEventLoop();
+        try {
+            final ServiceRequestContext ctx = spy(mock(ServiceRequestContext.class));
+            when(ctx.eventLoop()).thenReturn(ev);
 
-        final DefaultAttributeMap attrs = new DefaultAttributeMap();
-        attrs.attr(ROLE_MAP).set(roleMap::get);
-        when(ctx.attr(CURRENT_USER_KEY)).thenReturn(attrs.attr(CURRENT_USER_KEY));
-        when(ctx.attr(ROLE_MAP)).thenReturn(attrs.attr(ROLE_MAP));
+            final DefaultAttributeMap attrs = new DefaultAttributeMap();
+            attrs.attr(ROLE_MAP).set(roleMap::get);
+            when(ctx.attr(CURRENT_USER_KEY)).thenReturn(attrs.attr(CURRENT_USER_KEY));
+            when(ctx.attr(ROLE_MAP)).thenReturn(attrs.attr(ROLE_MAP));
 
-        try (SafeCloseable ignore = RequestContext.push(ctx)) {
+            try (SafeCloseable ignore = RequestContext.push(ctx)) {
 
-            attrs.attr(CURRENT_USER_KEY).set(User.DEFAULT);
+                attrs.attr(CURRENT_USER_KEY).set(User.DEFAULT);
 
-            when(ctx.pathParam("projectName")).thenReturn("A");
-            assertThat(new ProjectMembersOnly().serve(delegate, ctx, null).aggregate().join().status())
-                    .isEqualTo(HttpStatus.OK);
-            assertThat(new ProjectOwnersOnly().serve(delegate, ctx, null).aggregate().join().status())
-                    .isEqualTo(HttpStatus.OK);
+                when(ctx.pathParam("projectName")).thenReturn("A");
+                assertThat(new ProjectMembersOnly().serve(delegate, ctx, null).aggregate().join().status())
+                        .isEqualTo(HttpStatus.OK);
+                assertThat(new ProjectOwnersOnly().serve(delegate, ctx, null).aggregate().join().status())
+                        .isEqualTo(HttpStatus.OK);
 
-            when(ctx.pathParam("projectName")).thenReturn("B");
-            assertThat(new ProjectMembersOnly().serve(delegate, ctx, null).aggregate().join().status())
-                    .isEqualTo(HttpStatus.OK);
-            assertThatThrownBy(() -> new ProjectOwnersOnly().serve(delegate, ctx, null))
-                    .isInstanceOf(HttpStatusException.class)
-                    .satisfies(cause -> {
-                        assertThat(((HttpStatusException) cause).httpStatus())
-                                .isEqualTo(HttpStatus.UNAUTHORIZED);
-                    });
+                when(ctx.pathParam("projectName")).thenReturn("B");
+                assertThat(new ProjectMembersOnly().serve(delegate, ctx, null).aggregate().join().status())
+                        .isEqualTo(HttpStatus.OK);
+                assertThatThrownBy(() -> new ProjectOwnersOnly().serve(delegate, ctx, null))
+                        .isInstanceOf(HttpStatusException.class)
+                        .satisfies(cause -> {
+                            assertThat(((HttpStatusException) cause).httpStatus())
+                                    .isEqualTo(HttpStatus.UNAUTHORIZED);
+                        });
 
-            attrs.attr(CURRENT_USER_KEY).set(User.ADMIN);
+                attrs.attr(CURRENT_USER_KEY).set(User.ADMIN);
 
-            assertThat(new AdministratorsOnly().serve(delegate, ctx, null).aggregate().join().status())
-                    .isEqualTo(HttpStatus.OK);
-            assertThat(new ProjectMembersOnly().serve(delegate, ctx, null).aggregate().join().status())
-                    .isEqualTo(HttpStatus.OK);
-            assertThat(new ProjectOwnersOnly().serve(delegate, ctx, null).aggregate().join().status())
-                    .isEqualTo(HttpStatus.OK);
+                assertThat(new AdministratorsOnly().serve(delegate, ctx, null).aggregate().join().status())
+                        .isEqualTo(HttpStatus.OK);
+                assertThat(new ProjectMembersOnly().serve(delegate, ctx, null).aggregate().join().status())
+                        .isEqualTo(HttpStatus.OK);
+                assertThat(new ProjectOwnersOnly().serve(delegate, ctx, null).aggregate().join().status())
+                        .isEqualTo(HttpStatus.OK);
+            }
+        } finally {
+            ev.shutdownGracefully();
         }
     }
 }
```

**Comment by trustin** [comment]
Thanks, @huydx !

**Pull request merged**